src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +15,34 @@ public MovieDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
```

Comment:
Remember to use try with resources for Read operations. In this case, since you're working with a transaction, the existing try-finally block is appropriate, but make sure to use try with resources in the `get` method where you're only reading data.